docs/includes/net-standard-table.md

```diff
@@ -0,0 +1,26 @@
+| .NET Standard              | [1.0] | [1.1]  | [1.2] | [1.3] | [1.4] | [1.5]  | [1.6]  | [2.0] |
+|----------------------------|-------|--------|-------|-------|-------|--------|--------|-------|
+| .NET Core                  | 1.0   | 1.0    | 1.0   | 1.0   | 1.0   | 1.0    | 1.0    | 2.0   |
+| .NET Framework             | 4.5   | 4.5    | 4.5.1 | 4.6   | 4.6.1 | 4.6.2* | vNext* | 4.6.1 |
```

Comment:
@BillWagner what about this, are we added it?

@weshaggard said It's not necessary, but when I showed it to newbies, the above table is more clear to them than the one already there, with *'s confusing them.

I think It's better to tell the tooling explicitly!